tests/posix/fsstress.c

```diff
@@ -887,21 +897,25 @@ static int generate_fname(fent_t *fep, int ft, pathname_t *name, int *idp, int *
 
     flp = &flist[ft];
     len = RedSNPrintf(buf, sizeof(buf), "%c%x", flp->tag, id = nameseq++);
-    namerandpad(id, buf, len);
-    if (fep) {
-        fent_to_name(name, &flist[FT_DIR], fep);
-        append_pathname(name, "/");
-    }
-    append_pathname(name, buf);
-    *idp = id;
-    *v = verbose;
-    for (j = 0; !*v && j < ilistlen; j++) {
-        if (ilist[j] == id) {
-            *v = 1;
-            break;
+    if (len < 0) {
+        return 0;
+    } else {
```

Comment:
Putting the subsequent code inside an `else` statement is unnecessary, since the `if` statement returns.  Removing the `else` will eliminate the need to add indentation to the subsequent code in the function.